src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -14,15 +14,24 @@ type private Format = ReadOnlyMemory<byte>
 [<Struct>]
 type ExpectedVersion = Any | StreamVersion of int64
 
+module private Queries =
+    [<Literal>]
+    let writeMessage = "select * from write_message($1::text, $2, $3, $4, $5, $6)"
```

Comment:
🤔 any advantage in explicitly listing the desired returns?

Reply:
I think you're going to have to clarify what you mean here 🤔

Reply:
wondering if the `*` could/should be replaced with something more selective? (Doesn't really matter - I dont know about stored proc invocation syntax)

Reply:
Ah, I see now. IIRC the function returns a single bigint representing the new position of the stream (which we already know application side). Not sure if it was a named tuple that you can extract from or not 🤔

Reply:
forget I said anything ;)